frontend/ui/widget/menu.lua

```diff
@@ -596,6 +597,7 @@ local Menu = FocusManager:extend{
     -- if you want to embed the menu widget into another widget, set
     -- this to false
     is_popout = true,
+    title_bar_fm_style = nil, -- set to true to build increased title bar like in FM
```

Comment:
You can use `_fm_` in the key, but then states what it means unabreviated in the comment: `like in FileManager`.
(That is: don't emulate @NiLuJe :) and make it easy for future contributors.)